src/components/filterslider.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from 'react'
+import {clothingCategory,stylesByCategory,clothingColor,clothingSeason} from '../containers/api'
+import  '../styles/filterslider.css'
+
+
+export default class Filter extends Component {
+    constructor(props) {
+        super(props)
+        this.state = {
+            categories: clothingCategory,
+            chosencategory: '',
+            styles: [],
+            chosenstyle: '',
+            colors: clothingColor,
+            chosencolor: '',
+            seasons: clothingSeason,
+            chosenseason: ''
```

Comment:
I don't think that you need to have `clothingCategory`, `styles`, `clothingColor`, or `clothingSeason` in the component's internal state. Those are all constants so they are not usually considered state. State should be for things that are not constant and cannot be computed from props, e.g., user input, or something that happens as a result of user interaction with your component.